resources/policy-single-account-k8s-aws.json

```diff
@@ -0,0 +1,57 @@
+{
+  "Version": "<PUT_YOUR_OWN_VERSION>",
```

Comment:
for quickening copy-paste, we will put latest available version

```
<version_block> = "Version" : ("2008-10-17" | "2012-10-17")
```
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_grammar.html